vkconfig/mainwindow.cpp

```diff
@@ -731,12 +731,15 @@ void MainWindow::NewClicked() {
 void MainWindow::addCustomPaths() {
     SaveLastItem();
 
-    dlgCustomPaths dlg(this);
-    dlg.exec();
-
     // Get the tree state and clear it.
+    // This looks better athetically after the dialog
```

Comment:
athetically => aesthetically

(I think - maybe you mean "athletically" ;-)

Reply:
aesthetically
You are an eagle eyed man.